### ISSUE TITLE
Check for URL which lead nowhere or return 404 errors

### ISSUE THREAD

**Issue by jcowey** [DCLP/dclpxsltbox#189]
The following question was posed by @rla2118 in #187. I have separated it out in order to be able to close #187.
It looks like the links are being distributed correctly. I see, though, that the image links next to individual fragments at http://dclp.github.io/dclpxsltbox/output/dclp/63/62931.html are broken. Is this a problem on our end or on Berlin's?

**Comment by jcowey** [comment]
Not particularly urgent but could you, Carmen, use a script to return for us a list of non functioning links to images?

**Comment by HolgerEssler** [comment]
I learned that BerlPap have changed file names for their images. If I can get a list of all broken links in
`` <graphic> starting with url="http://ww2.smb.museum/berlpap/Original/" ``, they agreed to provide me with a concordance of the new links.

**Comment by paregorios** [comment]
@HolgerEssler has anyone been able to provide you with such a list yet?

**Comment by HolgerEssler** [comment]
@paregorios No, your question is the first reaction on my comment.

**Comment by paregorios** [comment]
OK @HolgerEssler I'll undertake to get you a list.

**Comment by paregorios** [comment]
@HolgerEssler 
Attached here please find a CSV file (zipped) in two columns: XML filename and Berlin image URL.
[berlin-gfx.csv.zip](https://github.com/DCLP/dclpxsltbox/files/652123/berlin-gfx.csv.zip)

**Comment by HolgerEssler** [comment]
Thanks for the list. I have sent it to the colleagues in Berlin and will
get back once I receive their data.

**Comment by HolgerEssler** [comment]
I had sent the list with the updated links to @paregorios and @Edelweiss on December 31. Perhaps it was lost during the holidays. Should I reemail it?

**Comment by paregorios** [comment]
Yes, the message had subject line "Links Berlin (issue #189)". I can forward again also if necessary. cc @jcowey

**Comment by paregorios** [comment]
This does not yet appear to have been fixed in the XML. @jcowey please determine whether this should and can be addressed before "Final Release" or whether it should be deferred to the "post-final projects" milestone.
